source/simulator/core.cc

```diff
@@ -695,7 +695,7 @@ namespace aspect
     // so we want to offset them by 128 and not allow more than 128 boundary ids.
     const unsigned int boundary_id_offset = 128;
     if (!boundary_temperature_manager.allows_fixed_temperature_on_outflow_boundaries())
-      replace_outflow_boundary_ids(boundary_id_offset);
+      replace_outflow_boundary_ids(boundary_id_offset, false, 0);
```

Comment:
You can even set the index to an invalid unsigned int here (since it is not going to be used in case of temperature).